terraform/mgmtcluster.tf

```diff
@@ -45,6 +49,13 @@ data "openstack_images_image_ids_v2" "images" {
   sort = "updated_at:desc"
 }
 
+resource "openstack_blockstorage_volume_v3" "mgmtcluster_volume" {
+  size        = 30
+  name        = "${var.prefix}-mgmtcluster"
```

Comment:
I guess I would have named it "${var.prefix}-mgmtcluster-root" but that is more a thing of taste.
(Yes I also suffix routers with -rtr or so ...)

Reply:
Well my personal taste is usually:

"<project>-<service>-<env>-<location>-<name>"

I did it mostly the same as the instance was, but I'm more then happy to comply here.